src/Components/Admins/Form/admins-form.module.css

```
@@ -0,0 +1,36 @@
+.sectionForm {
+    margin: 40px 0;
+}
+
+.form {
+    color: #373867;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    background-color: #DDD;
```

We're trying to be consistant with the css too
```suggestion
    background-color: #DDD;
    width: 400px;
```
the width is only a example, you could check what we did in the other components